src/Humanizer.Tests/ApiApprover/PublicApiApprovalTest.approve_public_api.approved.txt

```diff
@@ -1028,12 +1030,12 @@ namespace Humanizer.DateTimeHumanizeStrategy
     }
     public class PrecisionDateTimeHumanizeStrategy : Humanizer.DateTimeHumanizeStrategy.IDateTimeHumanizeStrategy
     {
-        public PrecisionDateTimeHumanizeStrategy(double precision = 0.75) { }
+        public PrecisionDateTimeHumanizeStrategy(double precision = 0,75) { }
```

Comment:
There is still no upstream version for this problem? :-( This line should be reverted, and 1038 also.